mobile_verifier/src/lib.rs

```diff
@@ -15,10 +15,11 @@ pub use settings::Settings;
 
 use async_trait::async_trait;
 
+use mobile_config::gateway_info::GatewayMetadata;
 pub enum GatewayResolution {
     GatewayNotFound,
     GatewayNotAsserted,
-    AssertedLocation(u64),
+    GatewayAsserted(GatewayMetadata),
```

Comment:
does it make sense to include the entire metadata struct in the `GatewayAsserted` variant when it should have metadata to include the device type even if it's not asserted?